elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/AsyncProperties.java

```diff
@@ -33,6 +33,11 @@
      */
     private int queryCleanupDays = 7;
 
+    /**
+     * Default cancel interval of async query.
+     */
+    private int queryCancellationSeconds = 300;
```

Comment:
I think the definition of this is vague.  What is this interval for?  Let's get a more descriptive name and a better description of its intended use.